package.json

```diff
@@ -60,6 +60,7 @@
     "mime": "^3.0.0",
     "node-emoji": "^1.11.0",
     "nyc": "^15.1.0",
+    "p-queue": "^6.6.2",
```

Comment:
Did yarn.lock not get updated and CI is not complaining?? 🧐

Reply:
It was already a dependency of matrix-appservice-bridge, a free one if you will :)